Week2/homework/js-exercises/5-who-wants-a-drink.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+// empty Array
+const drinkTray = [];
+
+// Array with three items
+const drinkTypes = ['cola', 'lemonade', 'water', ];
+
+
+for (let i = 0; i < 5; i++) {
+    let x = i % 3;
+    drinkTray.push(drinkTypes[x]);
+
+}
+// Output 
+console.log("Hey guys, I brought a " + drinkTray);
```

Comment:
This logic seems just right but it feels very unstable. For example, what happens if you have to iterate over the loop not 5 but 7 times. Then, we would have `cola` in it thrice, isn't it? This is because you don't have a check whether `drinkTray` already contains two instances of the item. 🤔 

But maybe for this exercise it is fine. 🙂

Reply:
thank you so much Yash-kapila for the feedback. I study strongly to develop myself in Javascript and i promise you next time to develop my homework.